samples/tfm_integration/config_build/sample.yaml

```diff
@@ -5,11 +5,12 @@ common:
   tags:
     - trusted-firmware-m
   platform_allow:
-    - mps2/an521/cpu0/ns
```

Comment:
Why removing that one?

Reply:
we are about to start treating overflow as errors and it does fail if used as integration platform also.